backend/de.metas.business/src/main/java/de/metas/pricing/service/impl/PriceListDAO.java

```diff
@@ -143,6 +143,7 @@ public PricingSystemId getPricingSystemIdByValueOrNull(@NonNull final String val
 				.createQueryBuilderOutOfTrx(I_M_PricingSystem.class)
 				.addOnlyActiveRecordsFilter()
 				.addEqualsFilter(I_M_PricingSystem.COLUMNNAME_Value, value)
+				.orderByDescending(I_M_PricingSystem.COLUMNNAME_AD_Client_ID)
 				.create()
 				.firstIdOnly();
```

Comment:
this will
```suggestion
				.firstId();
```

firstIdOnly woudl fail if there were >1 records